### ISSUE TITLE
[coqide] zoom in shortcut not working

### ISSUE THREAD

**Issue by Alizter** [coq/coq#14586]
#### Description of the problem

Doing `Ctrl + +` in coqide doesn't seem to work.

`Ctrl + -` does zoom out however.

#### Coq Version

8.15+alpha

**Comment by Alizter** [comment]
This seems to have always been an issue, going back through all the versions to coq 8.4. I seem to however recall that this worked at some point, this makes me suspect that there is an issue with the lablgtk library instead.

**Comment by Alizter** [comment]
It turns out that pressing `Ctrl + Shift + +` does zoom in. My keyboard is a UK keyboard so I don't usually press `Shift` to get `+`, whereas I believe US keyboards do. This seems like a keyboard locale issue.

**Comment by Alizter** [comment]
It's definitely standard behaviour on UK keyboards for a program to Zoom in with `Ctrl + =`. What is difficult here is that I could not work out how to bind both = and + to the Zoom action in the lablgtk bindings. I will leave this ticket open, but it is definitely low priority.

**Comment by jfehrle** [comment]
How do you change lablgtk bindings without a code change?  I've looked for this though maybe not hard enough, and it's something I want to describe in the Coq documentation.

**Comment by jfehrle** [comment]
So you're not aware of any way to rebind keys without a code change?

I expect `~accel` doesn't take multiple arguments because accelerators are shown in the menus:

![image](https://user-images.githubusercontent.com/1253341/170118640-c3c2d719-c168-45e5-8a0f-8351822a7273.png)

**Comment by Alizter** [comment]
>I expect ~accel doesn't take multiple arguments because accelerators are shown in the menus:

Exactly, but I don't know how else to add a shortcut. If there is API for this in lablgtk then wonderful, but I never found it.
